### ISSUE TITLE
Training and validation loss greater than 1

### ISSUE THREAD

**Issue by akshayk0406** [fchollet/keras#5718]
Hi,

I am sequential model for 3 class classification. When I am training the classifier i am seeing that training and validation loss are greater than 1. How is this possible. Can somebody please explain me ?

**Comment by joelthchao** [comment]
If you use categorical cross entropy (formula [here](https://wikimedia.org/api/rest_v1/media/math/render/svg/0cb6da032ab424eefdca0884cd4113fe578f4293)), it's possible to have loss greater than 1.
